guide/C/ch_basics.xml

```diff
@@ -625,144 +625,126 @@
+		formats. Users can choose a file format for new files from <menuchoice><guimenu>File</guimenu> 
+		<guimenuitem>Save </guimenuitem></menuchoice> and for existing files from <menuchoice><guimenu>File</guimenu> 
 		<guimenuitem>Save As...</guimenuitem></menuchoice> dialogs.</para>
 		
-		<para>The <acronym>XML</acronym> storage format is a text file that is compressed 
-		with gzip, which is a preference that is set at <menuchoice><guimenu>Edit</guimenu> 
+		<para>The <acronym>XML</acronym> storage format is a text file that by default is compressed, 
+		which is a preference that is set at <menuchoice><guimenu>Edit</guimenu> 
 		<guimenuitem>Preferences</guimenuitem></menuchoice> <guilabel>General</guilabel> 
-		<guilabel>Compress files</guilabel>. <acronym>SQL</acronym> storage is available 
-		via the <acronym>DBI</acronym> back end; use of MySQL and PostgreSQL requires 
-		installation of those <acronym>DBMS</acronym> packages as well as additional 
-		<acronym>DBI</acronym> drivers.</para>
+		<guilabel>Compress files</guilabel>. SQLite storage is also available, and stores 
+		your data in a single file on your system, like the <acronym>XML</acronym> format. However,
+		internally, an SQLite file is managed as a database. The MySQL and PostgreSQL storage options 
+		require access to a MySQL or PostgreSQL database server and the installation of 
+		additional database drivers on your machine.</para>
 		
 		<note>
 		<para>Users can change the format at any time by using <menuchoice><guimenu>File</guimenu> 
 		<guimenuitem>Save As...</guimenuitem></menuchoice>. This will create a copy of 
 		the data file in the selected format.</para>
 		</note>
-		<sect3 id="basics-files-storage-comparison">
-			<title>Storage Comparison</title>
-			
-			<para>Each storage format has benefits and shortcomings that users should 
-			consider for their needs and abilities. See the Storage Comparison Table below 
-			for further details.</para>
-			
-			<para>The <acronym>XML</acronym> format is the most stable and established, 
-			and for this reason, it is recommended for most users. <acronym>SQL</acronym>
-			storage was added for the 2.4 release and has become an increasingly popular 
-			choice for users, but can require experience with <acronym>DBMS</acronym> 
-			applications.</para>
+
+	<sect2 id="basics-files-storage-comparison">
+		<title>Storage Comparison and Recommendations</title>
+		<para>Each storage format has benefits and shortcomings that users should 
+		consider for their needs and abilities. See the Storage Comparison Table below 
+		for further details.</para>
+		
+		<para>The <acronym>XML</acronym> format is the most stable and established, 
+		and for this reason, it is recommended for most users. <acronym>SQL</acronym>
+		storage was added for the 2.4 release and has become an increasingly popular 
+		choice for users, but can require experience with <acronym>DBMS</acronym> 
+		applications.</para>
```

Comment:
I think it would make sense to make a distinction here between Sqlite3 and MySQL/PostgreSQL to clarify where the experience with DBMS applications pops in.
Especially since we as a project want to guide users deliberately to sqlite as the preferred sql option. Only users with sufficient DBMS experience should be suggested to use the MySQL or PostgreSQL options.

I'm aware this is also contained in your table, but as I said before the "manager level" decision will most likely already be taken in this section.

Reply:
How about:

SQL storage was added for the 2.4 release and has become an increasingly popular choice for users. The SQLite format allows users to realize the benefits of SQL storage without the overhead of installing or managing a full DBMS.  MySQL and PostgreSQL require the installation of MySQL and PostgreSQL, respectively, and are best implemented only by experienced database administrators.

Reply:
Good enough for me.

Going pedantic one could argue that for MySQL and PostgreSQL one only needs an account on some MySQL or PostgreSQL server (which can also be a hosted service). However users sufficiently experienced with either DB will perfectly understand this from how you have formulated it as well.